sql/core/src/test/resources/sql-tests/inputs/explain.sql

```diff
@@ -0,0 +1,97 @@
+-- Test tables
+CREATE table  explain_temp1 (key int, val int) USING PARQUET;
+CREATE table  explain_temp2 (key int, val int) USING PARQUET;
+CREATE table  explain_temp3 (key int, val int) USING PARQUET;
+
+-- Simple explain
```

Comment:
what does this comment mean?

Reply:
@cloud-fan not much actually :-). I will remove.